source/dbglogger.c

```diff
@@ -258,7 +258,10 @@ void dbglogger_log(const char* fmt, ...) {
         va_start(arg, fmt);
         vsnprintf(buffer, sizeof(buffer), fmt, arg);
         va_end(arg);
-    
+
+#ifdef __PS4__
+        puts(buffer);
+#endif

```

Comment:
let's avoid this and add a `TTY_LOGGER` output if needed.

Reply:
Added `__USE_TTY_LOG__`